epicfreegame/index.js

```diff
@@ -119,13 +130,9 @@ async function unsub() {
 async function registerServiceWorker() {
 	// check if service worker has been registered
 	let registration = await navigator.serviceWorker.getRegistration(serviceWorker)
-	if (registration) {
-		// try to update the service worker
-		await registration.update()
-	} else {
+	if (!registration) {
 		registration = await navigator.serviceWorker.register(serviceWorker, {
-			type: "module",
-			updateViaCache: "all"
```

Comment:
The default behaviour is `"imports"`: https://www.w3.org/TR/service-workers/#dfn-update-via-cache